backend/src/main/java/ai/verta/modeldb/configuration/MigrationsIncludedInAppStartup.java

```diff
@@ -13,14 +13,14 @@
 import org.springframework.context.annotation.ConditionContext;
 import org.springframework.core.type.AnnotatedTypeMetadata;
 
-public class MigrationSetupConfig implements Condition {
-  private static final Logger LOGGER = LogManager.getLogger(MigrationSetupConfig.class);
+public class MigrationsIncludedInAppStartup implements Condition {
```

Comment:
Heh. I think my intention was to separate the condition from the config, and have 2 different beans, rather than just renaming the config bean.

Reply:
Fix trying to implement more clearly using separate beans